src/targets/gpu/CMakeLists.txt

```diff
@@ -46,6 +49,15 @@ else()
     message(STATUS "MIGraphX build without rocBLAS")
 endif()
 
+if(MIGRAPHX_USE_HIPBLASLT)
+    # hipblaslt
+    find_package(hipblaslt REQUIRED)
+    find_package(hipblas REQUIRED)
```

Comment:
Why are we looking for hipblas? We dont use it.

Reply:
Removed.

Reply:
Seems like hipblaslt uses hipblas. I saw sles failure after removing hipblas:
```
/opt/rocm-6.1.3/include/hipblaslt/hipblaslt.h:49:10: fatal error: 'hipblas/hipblas.h' file not found
   49 | #include <hipblas/hipblas.h>
      |          ^~~~~~~~~~~~~~~~~~~
```

Reply:
This seems to be causing this error too:
```
File "/src/AMDMIGraphX/tools/accuracy/accuracy_checker.py", line 26, in
import migraphx
ImportError: /src/AMDMIGraphX/build/lib/libmigraphx_gpu.so.2011000: undefined symbol: hipblasSetStream
```

Reply:
Ahsan to put a comment to add that we are doing this to workaround the broken hipblaslt package.
To Do:
1. Verify with hipBLASLt team if this is not fixed and open an issue to fix the hipblaslt package:
   a. Dev package needs to be fixed to inclue hipblas header files.
   b. Automatically add hipblas in cmake.

Reply:
Added comment to explain its a workaround.

Reply:
Chris checked and the answer to the broken issue is:
```
In 6.2, hipBLAS is a dependency of hipBLASLt, so you should be able to use them together.
In 6.3, the dependencies got change around a bit and you will need to also install a package
called hipBLAS-common.
When you are adding your code now, please don’t assume that just because you have
hipBLASlt installed that means you will have hipBLAS, since this assumption will go away in 6.3.
If you need hipBLAS, install it explicitly and #include it explicitly.
```
This explains how the code in develop that we were looking at would work. They have a new package hipBLAS-common.
```
#ifndef LEGACY_HIPBLAS_DIRECT
#include <hipblas-common/hipblas-common.h>
#else
#include <hipblas/hipblas.h>
#endif
```
@pfultz2 In light of above, do we want to keep hipblas for now since CI is at 6.1. Once we update the CI to 6.3, we can remove hipblas requirement and add `hipBLAS-common` requirement. Does this seem a reasonable approach for now?

Reply:
I missed the step of updating CI to 6.2 before we update it to 6.3 :). I imagine that 6.2 should work fine though (with current workaround), since the changes for hipblaslt side went in, in 6.3.

Reply:
Yea we need a comment for the workaround since there package is broken in 6.1(and possibly 6.2 as well).

If hipBLAS-common package is needed in 6.3, then they need to make sure its included by default. We are not using hipblas at all.